윤건수/programmers/LV2_42587/Solution.java

```diff
@@ -0,0 +1,53 @@
+package programmers.LV2_42587;
+
+import java.util.*;
+
+public class Solution {
+    public int solution(int[] priorities, int location) {
+        PriorityQueue<Priority> sortQueue = new PriorityQueue<>();
+        for(int i = 0; i < priorities.length; i++){
+            sortQueue.add(new Priority(i, priorities[i]));
+        }
+
+        Queue<Priority> queue = new LinkedList();
+        for(int i = 0; i < priorities.length; i++){
+            queue.add(new Priority(i, priorities[i]));
+        }
```

Comment:
```suggestion
        PriorityQueue<Priority> sortQueue = new PriorityQueue<>();
        Queue<Priority> queue = new LinkedList();
        
        for(int i = 0; i < priorities.length; i++){
            Priority priority = new Priority(i, priorities[i]);

            sortQueue.add(priority);
            queue.add(priority);
        }
```

for 루프 하나로도 해결 가능해 보입니다.